base/ca/src/org/dogtagpki/ct/sct/SCTProcessor.java

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+
+package org.dogtagpki.ct.sct;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.dogtagpki.ct.LogServer;
+import org.dogtagpki.server.ca.CAEngine;
+import org.dogtagpki.server.ca.CAEngineConfig;
+
+import com.netscape.certsrv.base.EBaseException;
+import com.netscape.certsrv.base.EPropertyNotFound;
+import com.netscape.certsrv.base.IConfigStore;
+
+/**
+ * @author Dinesh Prasanth M K
+ *
+ */
+public class SCTProcessor {
+    public static org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(SCTProcessor.class);
+
+    protected IConfigStore mConfig;
+
+    public void init() throws Exception {
+        CAEngine engine = CAEngine.getInstance();
+        CAEngineConfig cs = engine.getConfig();
+
+        mConfig = cs.getCAConfig().getSubStore("certTransparency");
+    }
+
+    public boolean isCTEnabled() throws EPropertyNotFound, EBaseException {
+        return mConfig.getBoolean("enable");
+    }
+
+    public List<LogServer> getLogServerConfig() throws EPropertyNotFound, EBaseException, MalformedURLException {
+
```

Comment:
perhaps for clarity, add a comment with an example set of CS.cfg's CT log config.

Reply:
Done